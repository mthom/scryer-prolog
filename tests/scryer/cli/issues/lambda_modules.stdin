use_module(a).
make_goal_a(G), call(G).
make_goal_b_raw(G), call(G).
make_goal_b_raw(G), call(b:G).
make_goal_b_scoped(G), call(G).
make_goal_b_scoped_alt(G), call(G).
