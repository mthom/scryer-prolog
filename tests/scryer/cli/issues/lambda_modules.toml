# issue 2255
args = ["-f", "--no-add-history"]
