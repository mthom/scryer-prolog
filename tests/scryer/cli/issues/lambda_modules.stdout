   true.
   G = /p.
   error(existence_error(procedure,q/0),q/0).
   G = /q.
   G = / (b:q).
   G = b: /q.
